src/account/default.ts

```diff
@@ -122,13 +144,32 @@ export class Account extends Provider implements AccountInterface {
     return this.cairoVersion;
   }
 
+  /**
+   * Calculates the estimated fee for executing a series of function calls.
+   *
+   * @param {AllowArray<Call>} calls - The list of function calls to estimate the fee for.
+   * @param {UniversalDetails} [estimateFeeDetails = {}] - Additional details for fee estimation. Default is an empty object.
+   * @returns {Promise<EstimateFee>} - A promise that resolves to the estimated fee.
+   */
   public async estimateFee(
     calls: AllowArray<Call>,
     estimateFeeDetails: UniversalDetails = {}
   ): Promise<EstimateFee> {
     return this.estimateInvokeFee(calls, estimateFeeDetails);
   }
 
+  /**
+   * Estimate the invoke fee for a given set of calls.
```

Comment:
sorry for forgetting about mentioning this, but the interface (/account/interface.ts) has docs for these [methods](https://github.com/starknet-io/starknet.js/blob/b16db6db7120ba57a8dab13516a8ccb0ad9f3c78/src/account/interface.ts#L42)
...maybe not for all of them, so for the missing ones I would add docs, and remove where they are already present in the interface.
The same thing should be done for `contract.ts` and `signer.ts`